admin-dev/themes/new-theme/js/pages/product/components/combination-list/bulk-edition-handler.ts

```diff
@@ -146,21 +151,20 @@ export default class BulkEditionHandler {
 
   private async submitForm(form: HTMLFormElement): Promise<void> {
     const progressModal = this.showProgressModal();
-
-    const checkboxes = this.bulkChoicesSelector.getSelectedCheckboxes();
+    const selectedIds = await this.bulkChoicesSelector.getSelectedIds();
     const progressModalElement = document.getElementById(CombinationMap.bulkProgressModalId);
 
     let progress = 1;
 
-    for (let i = 0; i < checkboxes.length; i += 1) {
-      const checkbox = checkboxes[i];
+    for (let i = 0; i < selectedIds.length; i += 1) {
+      const combinationId = selectedIds[i];
 
       // @todo when the ProgressModal will be integrated this will update it after each request
       try {
         // eslint-disable-next-line no-await-in-loop
         const response: Response = await this.combinationsService.bulkUpdate(
```

Comment:
Same thing there, you could probably manage it using `Promise.all`

Reply:
No we want to wait on purpose as explained in the bulk delete component